src/stim/py/numpy.pybind.h

```diff
@@ -31,6 +31,9 @@ pybind11::object simd_bits_to_numpy(
     stim::simd_bits_range_ref<stim::MAX_BITWORD_WIDTH> bits, size_t num_bits, bool bit_packed);
 void memcpy_bits_from_numpy_to_simd(
     size_t num_bits, const pybind11::object &src, stim::simd_bits_range_ref<stim::MAX_BITWORD_WIDTH> dst);
+size_t numpy_to_size(const pybind11::object &numpy_array, size_t expected_size);
```

Comment:
Since these are moving into a header, make the name longer and clearer like `bit_count_of_bit_packable_numpy_array` or something.